add support for Huion 0x006f (H950P T204)

tequeter opened the pull request:
I've got this H950P with a new ProductId `0x006f`. The strings are as follows:
* Product: `Huion Tablet_H950P`
* Firmware (string descriptor 201): `HUION_T204_230821`

This patch makes the buttons work in `evtest` and consequently in `xsetwacom`. They do not work in KDE's KCM, but that's another issue.

Stylus X/Y tilting also works. The resolution seems poor, but it might just be the hardware. I would have to try tilting with the proprietary driver to find out.

There are too many input devices created for the actual capabilities of the H950P, but that's probably a feature in case a new `0x006f` model makes use of them. Of the 5 devices below, the last 3 are useless AFAIK:

```
/dev/input/event10:     HUION Huion Tablet_H950P
/dev/input/event11:     HUION Huion Tablet_H950P Pad
/dev/input/event12:     HUION Huion Tablet_H950P Touch Strip
/dev/input/event13:     HUION Huion Tablet_H950P Dial
/dev/input/event14:     HUION Huion Tablet_H950P
```

And lastly, I skipped `USB_DEVICE_ID_HUION_TABLET3` as it is claimed by #623 and #644 already.

JoseExposito commented:
Hi @tequeter,

I plan to create a DIGImend release (v13) this weekend with the changes already in master and I'd like to include the new product ID for your tablet (0x006f) as well as https://github.com/DIGImend/digimend-kernel-drivers/pull/685 (0x0064).

This should allow us to get some actual testing (and bug reports) before we send the patches to the upstream kernel.

The upstream kernel requires a `Signed-off-by: Name <your@email.com>` line at the end of the commit message. Could you add it, please? Or just add a comment and I'll add it to your commit.

Thanks!

tequeter commented:
Hello @JoseExposito,

Great! I rebased, and updated the commit with:

- `HID: uclogic:` prefix.
- Link to this PR.
- Sign-off.

Do you want to keep the `USB_DEVICE_ID_HUION_TABLET3` gap?

NB: I do not own the H950P anymore, I returned it for a H1061P (#679) instead.

Thomas

JoseExposito reviewed (approved):
Thanks a lot! Merging :tada:
